app/views/world_location_news/index.html.erb

```diff
@@ -3,21 +3,27 @@
 <% atom_discovery_link_tag atom_feed_url_for(@world_location), t("feeds.latest_activity") %>
 
 <%= content_tag_for(:article, @world_location) do %>
-  <header class="block headings-block">
-    <div class="inner-block floated-children">
-      <%= render partial: 'shared/heading',
-                locals: { type: "World location news",
-                          heading: @world_location.title,
-                          extra: true, big: true } %>
-      <aside class="heading-extra">
-        <div class="inner-heading">
-          <%= render 'shared/available_languages', object: @world_location %>
-          <% lang = "lang=en" unless I18n.locale.eql?(:en) %>
-          <%= render 'shared/featured_links', links: @world_location.featured_links.only_the_initial_set, lang: lang %>
+  <div class="govuk-width-container">
+    <div class="govuk-grid-row">
+      <div class="govuk-grid-column-two-thirds">
+        <%= render "govuk_publishing_components/components/title", {
+          context: "World location news",
+          title: @world_location.title,
+        } %>
+      </div>
+      <div class="govuk-grid-column-one-third">
+        <div class="govuk-!-margin-top-8 govuk-!-margin-bottom-8">
+          <aside class="heading-extra">
+            <div class="inner-heading">
+              <%= render 'shared/available_languages', object: @world_location %>
```

Comment:
Another out of scope comment: It looks like this is rendering a custom translation nav as well as scoping the decision to render it to that component, which I personally don't like as it makes it harder to understand at a glance if the component is rendering or not. Could we replace this with a [translation nav component](https://components.publishing.service.gov.uk/component-guide/translation_nav)?